turbinia/api/routes/config.py

```diff
@@ -58,3 +59,16 @@ async def get_version(request: Request):
   """Returns the Turbinia version."""
   response = {'version': __version__}
   return JSONResponse(content=response, status_code=200)
+
+
+@router.get('/download')
+async def download_config(request: Request):
+  """Downloads Turbinia configuration."""
+  try:
+    content = api_utils.get_configuration_content()
+    return Response(content=content, status_code=200)
```

Comment:
This should work. However, FastAPI provides a ```FileResponse``` type you can use. It should simplify the code quite a bit. https://fastapi.tiangolo.com/advanced/custom-response/#fileresponse

Reply:
Here is an example for a unit test. The client is already prepared and you just have to call self.client.get(...) and compare the response to what's expected.

https://github.com/google/turbinia/blob/master/turbinia/api/api_server_test.py#L206